vars/artifactsPipeline.groovy

```diff
@@ -84,8 +84,6 @@ def call(Map pipelineParams) {
         }
         options {
             timestamps()
-            // Timeout for the whole test, add 30 MINUTES for waiting the builder
-            timeout([time: pipelineParams.timeout.time + 30, unit: 'MINUTES'])
```

Comment:
why removing completely? why not increasing it to 10 hours for instance?

Reply:
in any of the other pipeline we removed the total timeout, since we set timeout for each specific step, there's no need for one more
also this timeout is counting the waiting for resources, which we don't want.